README.md

```diff
@@ -1,25 +1,24 @@
-# SEO: A plugin for Pelican :fr:
+# SEO Plugin for Pelican :fr:
 
 [![Build Status](https://img.shields.io/github/workflow/status/pelican-plugins/seo/build)](https://github.com/pelican-plugins/seo/actions)
 
-This plugin helps you improve your Pelican site SEO (Search Engine Optimization) to reach the top positions on search engines. To see what can do SEO for you, go directly to [Usage](#usage) section.
+This plugin helps you improve your Pelican site SEO (Search Engine Optimization) to reach the top positions on search engines. To see what this SEO plugin can do for you, visit the [Usage](#usage) section.
 
 | Author          | GitHub                            |
 | :-------------: | :-------------------------------: |
 | Maëva Brunelles | https://github.com/MaevaBrunelles |
 
-## Why do you need SEO
+## Why
```

Comment:
I don't think it's a user-friendly title (neither a SEO-friendly title :)).
We can update it for `Why you might want to use this plugin` if you prefer, and keep the initial `If:`

Reply:
How about: `## Why Use This Plugin?`

Reply:
Agreed